config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  default_url_options :host => 'localhost:3000' # this code-line is for request tests
```

Comment:
is it a good idea to write a specific thing in the routes.rb file for tests or for dev?
Or maybe there are other solutions?
are there any files that change the config just for testing purposes?

Reply:
i have problems with request spec for devise token auth, and it's only one solution which helped me, i try to improve this for configuration file

Reply:
config/environments/test.rb file?)